.github/workflows/ci.yml

```
@@ -42,6 +42,8 @@ jobs:
           pip install -r requirements/dev.txt
           pip check
   test:
+    # Time out after 10 minutes if our test suite has gotten hung
+    timeout-minutes: 10
```

```suggestion
    # Time out if our test suite has gotten hung
    timeout-minutes: 15
```

Give is a little more wiggle room, and remove the comment specificity